src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudSnapshotCreatedIT.java

```diff
@@ -64,7 +64,7 @@
 
   private static final int SNAPSHOT_TEST_TIMEOUT = 120;
 
-  @ClassRule public static Timeout timeout = new Timeout(5, TimeUnit.MINUTES);
+  @ClassRule public static Timeout timeout = new Timeout(20, TimeUnit.MINUTES);
```

Comment:
Is a 20 minute timeout necessary for this? Snapshot creation shouldn't be taking that long.

Reply:
When I tested this on windows it would timeout even with a timeout as high as 15 minutes, and given that this particular test takes twice as long as the non-snapshot based tests on average, it seemed reasonable in context. Snapshot creation is the longest test in this suite. There's a lot more contention for threads and ram with the windows host I used than with my local machine. I'm going to try repeating my tests with an ssh connection as opposed to RDP and see if things speed up then.

Reply:
That seems odd, as all that's been done is issuing REST requests and waiting on responses. We'll need to take a closer look to see why this is happening and how we might be able to mitigate it.

Reply:
Windows snapshots take a long time to create from my experience.